Sources/NIO/SelectableEventLoop.swift

```diff
@@ -179,7 +179,11 @@ Further information:
         self.tasksCopy.reserveCapacity(4096)
         self.canBeShutdownIndividually = canBeShutdownIndividually
         // note: We are creating a reference cycle here that we'll break when shutting the SelectableEventLoop down.
-        self._succeededVoidFuture = EventLoopFuture(eventLoop: self, value: (), file: "n/a", line: 0)
+        // note: We have to create the promise and complete it because otherwise we'll hit a loop in `makeSucceededFuture`. This is
```

Comment:
Is this worth filing an issue about?

Reply:
I don't think so: this is an unfortunate wrinkle in the way the fast-path for completed void futures works. It's kinda unavoidable now that we've split them apart, and there isn't a good way to fix this.